sysinfo.go

```diff
@@ -32,13 +35,51 @@ const (
 	ProcessorArchitectureAMD64   ProcessorArchitecture = wrappers.PROCESSOR_ARCHITECTURE_AMD64
 )
 
+type DisplayDevice struct {
+	DeviceName   string
+	DeviceString string
+	StateFlags   uint32
+	DeviceID     string
+	DeviceKey    string
+}
+
+type DisplayMonitorInfo struct {
+	Handle wrappers.HMONITOR
+	Hdc    wrappers.HDC
```

Comment:
Since we don't use Hungarian notation on the Go side, let's rename this field to `DeviceContext`.